exchanges/refocus/package.json

```diff
@@ -56,7 +56,7 @@
     "graphql": "^0.11.0 || ^0.12.0 || ^0.13.0 || ^14.0.0 || ^15.0.0 || ^16.0.0"
   },
   "dependencies": {
-    "@urql/core": ">=3.2.2",
+    "@urql/core": ">=4.0.0",
```

Comment:
Looks like `updateInternalDependents.updateInternalDependents` is failing and has done so for a while. I have no clue why it thinks it should update these dependencies when they're clearly in range of the upgrade 🤔